.github/workflows/update-make-docs.yml

```diff
@@ -3,12 +3,18 @@ on:
   schedule:
     - cron: '0 7 * * 1-5'
   workflow_dispatch:
+
+permissions:
+  contents: read
+
 jobs:
   main:
     if: github.repository == 'grafana/k6-docs'
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
       - uses: grafana/writers-toolkit/update-make-docs@update-make-docs/v1
```

Comment:
Pretty sure this will need contest:write given that it needs to update the repo

Reply:
I wasn't sure, but I'll update it 👍